ios/PolyPodApp/FeatureContainer/FeatureContainerView.swift

```diff
@@ -87,17 +147,25 @@ class FeatureWebView: WKWebView {
         let configuration = WKWebViewConfiguration()
         configuration.userContentController = contentController
 
+        let scheme = PolyNav.fsPrefix.replacingOccurrences(of: "://", with: "")
+        let fileHandler = FeatureFileHandler()
+        fileHandler.setFeature(feature: feature)
+        configuration.setURLSchemeHandler(fileHandler, forURLScheme: scheme)
+
         super.init(frame: .zero, configuration: configuration)
+
         scrollView.isScrollEnabled = false
         translatesAutoresizingMaskIntoConstraints = false
         MessageName.allCases.forEach {
             contentController.add(self, name: $0.rawValue)
         }
         removeInputAccessory()
 
-        let featureUrl = feature.path
-        let featureFileUrl = featureUrl.appendingPathComponent("pod.html")
-        loadFileURL(featureFileUrl, allowingReadAccessTo: featureUrl)
+        var components = URLComponents()
+        components.scheme = scheme
+        components.path = "/pod.html"
```

Comment:
This is always going to have the same name? Maybe create a constant? (also, maybe at the polypod level)